test/generator/autograd_aottest.cpp

```diff
@@ -9,7 +9,7 @@
 
 using namespace Halide::Runtime;
 
-const int kSize = 32;
+constexpr int kSize = 64;
```

Comment:
At global scope is constexpr int meaningfully different to const int? Any reason to prefer it?

Reply:
Not really, it's just reflexive change at this point. Want me to back it out?

Reply:
Was mostly wondering if we should make this change more broadly as a matter of style. The internet seems to suggest that constexpr is very slightly preferable because it guarantees you can use it in other compile-time constructs like static_assert. I think uniformity of style is a good thing, and this seems like mostly a stylistic choice, but I don't care enough to delay a PR over it, so lgtm.